projects/angular-calendar/schematics/utils/config.ts

```diff
@@ -0,0 +1,53 @@
+import { Tree, SchematicsException } from '@angular-devkit/schematics';
+
+const CONFIG_PATH = 'angular.json';
+
+export function addStyle(host: Tree, stylePath: string): void {
+  const config = readConfig(host);
+  const appConfig = getAngularAppConfig(config);
+
+  if (appConfig) {
+    appConfig.architect.build.options.styles.push({
+      input: stylePath
+    });
+    appConfig.architect.test.options.styles.push({
+      input: stylePath
+    });
+
+    writeConfig(host, config);
+  } else {
+    throw new SchematicsException(`app configuration could not be found`);
+  }
+}
+
+function readConfig(host: Tree): JSON {
+  const sourceText = host.read(CONFIG_PATH)!.toString('utf-8');
+  return JSON.parse(sourceText);
+}
+
+function writeConfig(host: Tree, config: JSON): void {
+  host.overwrite(CONFIG_PATH, JSON.stringify(config, null, 2));
+}
+
+function isAngularBrowserProject(projectConfig: any) {
+  if (projectConfig.projectType === 'application') {
+    const buildConfig = projectConfig.architect.build;
+    return buildConfig.builder === '@angular-devkit/build-angular:browser';
+  }
+
+  return false;
+}
+
+function getAngularAppConfig(config: any): any | null {
```

Comment:
Can we add a new option `projectName` to the `schema.json` to let the user choose the project to add the styles to. and then if none is provided use the project in `config.defaultProject`

Reply:
I'll take a look at this using multiple workspace projects

Reply:
done